pkg/apis/azuredisk/v1alpha1/types.go

```diff
@@ -29,12 +29,13 @@ type AzVolume struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
 	Spec   AzVolumeSpec   `json:"spec"`
-	Status AzVolumeStatus `json:"status"`
+	Status AzVolumeStatus `json:"status,omitempty"`
```

Comment:
`Status` should be marked `optional` and be a pointer type, i.e. `Status *AzVolumeStatus`.

Reply:
Are we going to be using pointer status for AzVolume object as well?

Reply:
Done :)